config/users/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from . import views
+
+app_name = 'users'
+
+urlpatterns = [
+    path('sign_up/', views.sign_up_view, name='sign_up'),
+    path('sign_in/', views.sign_in_view, name='sign_in'),
+    path('sign_out/', views.sign_out_view, name='sign_out'),
+    path('user_pets/', views.user_pets_list, name='user_pets_list'),
```

Comment:
to trochę bardziej pasuje już do aplikacji `pets` niż `users`

Reply:
'user_pets_list' moved to pets